crates/common/src/lib.rs

```diff
@@ -0,0 +1,48 @@
+
```

Comment:
this crate can disable std, see

https://github.com/alloy-rs/eips/blob/87943065cd04455e6365fe44e625e34f972cc1db/crates/eip2930/src/lib.rs#L4-L4

Reply:
i wonder, what platforms it could be disabled on